--- libcss-0.5.0/Makefile.orig	2015-03-16 11:57:17.844943636 +0100
+++ libcss-0.5.0/Makefile	2015-03-16 11:58:12.197982514 +0100
@@ -15,12 +15,7 @@ TESTRUNNER := $(PERL) $(NSTESTTOOLS)/tes
 WARNFLAGS := -Wall -W -Wundef -Wpointer-arith -Wcast-align \
 	-Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes \
 	-Wmissing-declarations -Wnested-externs
-# BeOS/Haiku/AmigaOS4 standard library headers create warnings
-ifneq ($(HOST),i586-pc-haiku)
-  ifneq ($(findstring amigaos,$(HOST)),amigaos)
-    WARNFLAGS := $(WARNFLAGS) -Werror
-  endif
-endif
+
 CFLAGS := -D_BSD_SOURCE -I$(CURDIR)/include/ \
 	-I$(CURDIR)/src $(WARNFLAGS) $(CFLAGS)
 ifneq ($(GCCVER),2)
